src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java

```diff
@@ -228,6 +232,16 @@ public void cliccareSuContinua() {
         invioNotifichePAPage.selezionareContinuaButton();
     }
 
+    @And("Aspetta {int} secondi")
+    public void aspettaSecondi(int quantiSecondi) {
+        logger.info("Aspetta " + quantiSecondi + " secondi");
+        try {
+            TimeUnit.SECONDS.sleep(quantiSecondi);
+        } catch (Exception exc) {
+            throw new RuntimeException(exc);
```

Comment:
insert log when entering catch

Reply:
I added a log, please check that there were already seven instances of `try {...} catch (...) {...}` inside this class, and none of them includes a log, just re-throws inside the envelop of a `RuntimeException`.